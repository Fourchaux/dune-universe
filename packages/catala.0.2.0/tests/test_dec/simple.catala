@Article@

/*
new scope A:
  param x content dec 
  param y content dec 
  param z content dec

scope A:
  def x := 84.648665
  def y := 4.368297
  def z := x /. y 
*/