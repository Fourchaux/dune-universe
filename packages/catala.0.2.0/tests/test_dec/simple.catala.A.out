[RESULT] Computation successful! Results:
[RESULT] x = 84.0648665
[RESULT] y = 4.368297
[RESULT] z = 19.24431111254569000230â€¦
